recipe/meta.yaml

```
@@ -32,9 +38,10 @@ test:
     - conda_auth
   commands:
     - pip check
-    - conda auth --help
-    - conda auth login --help
-    - conda auth logout --help
+    # As with all conda plugins, this must be installed into the base environment
+    #- conda auth --help
+    #- conda auth login --help
+    #- conda auth logout --help
```

```suggestion
    - python -m conda auth --help
    - python -m conda auth login --help
    - python -m conda auth logout --help
```

@jezdez, I think this should do it. I tested it on Windows, and it worked for me. Pretty sure this will run smoothly on other platforms too.

---

Thanks, that works.